integration-libs/opf/components/opf-checkout-payment-and-review/opf-checkout-payment-and-review.module.ts

```diff
@@ -25,6 +39,7 @@ import { OpfCheckoutPaymentsModule } from '../opf-checkout-payments/opf-checkout
           component: OPFCheckoutPaymentAndReviewComponent,
           guards: [CheckoutAuthGuard, CartNotEmptyGuard],
         },
+        CheckoutPlaceOrder: {},
```

Comment:
`could:` maybe it's worth consulting this with some architect even at this stage of implementation?

Reply:
After discussing the technical difficulty of this matter, it will be done in separate task / pull request